Strategic-Initiatives.md

```diff
@@ -11,20 +11,130 @@ and have the support needed.
-| http2             | James Snell                     |                                                                  |
-| Modules           | Myles Borins                    | https://github.com/nodejs/node-eps/blob/master/002-es-modules.md |
-| N-API             | Michael Dawson                  | https://github.com/nodejs/abi-stable-node                        |
-| OpenSSL Evolution | Rod Vagg                        | https://github.com/nodejs/TSC/issues/364                         |
-| Workers           | Anna Henningson                 | https://github.com/nodejs/worker                                 |
-| Error Messages    | James Snell                     | https://github.com/nodejs/node/issues/11273                      |
-| Core Promise APIs | James Snell                     |                                                                  |
-| TSC Governance    | Myles Borins                    | https://github.com/nodejs/TSC/issues/383                         |
-| New Streams APIs  | James Snell + Jeremiah Senkpiel | https://github.com/nodejs/node/pull/16414                        |
-| Moderation        | Rich Trott                      |                                                                  |
-| Async Hooks       | Trevor Norris                   |                                                                  |
-| V8 Currency       | Michaël Zasso                   |                                                                  |
+* [http2](#http2)
+* [Modules](#modules)
+* [N-API](#n-api)
+* [OpenSSL Evolution](#openssl-evolution)
+* [Workers](#workers)
+* [Error Messages](#error-messages)
+* [Core Promise APIs](#core-promise-apis)
+* [TSC Governance](#tsc-governance)
+* [New Streams APIs](#new-streams-apis)
+* [Moderation](#moderation)
+* [Async Hooks](#async-hooks)
+* [V8 Currency](#v8-currency)
+
+## http2
+
+### Champion
+
+[James Snell][jasnell]
+
+### Stakeholders
+
+[Matteo Collina][mcollina]
+
+## Modules
+
+### Champion
+
+[Myles Borins][MylesBorins]
+
+### Stakeholders
+
+[Bradley Farias][bmeck]
+[Guy Bedford][guybedford]
+[Jan Olaf Krems][jkrems]
+[Michaël Zasso][targos]
+
+### Important links
+
+* [node-eps for esm](https://github.com/nodejs/node-eps/blob/master/002-es-modules.md)
+* [the current state of implementation and planning for esmodules](https://medium.com/the-node-js-collection/the-current-state-of-implementation-and-planning-for-esmodules-a4ecb2aac07a)
+
+## N-API
+
+### Champion
+
+[Michael Dawson][mhdawson]
+
+### Important Links
+
+* [abi-stable-node repo](https://github.com/nodejs/abi-stable-node)
+
+## OpenSSL Evolution
+
+### Champion
+
+[Rod Vagg][rvagg]
+
+### Important Links
+
+* [tracking issue on TSC repo](https://github.com/nodejs/TSC/issues/364)
+
+## Workers
+
+### Champion
+
+[Anna Henningson][addaleax]
+
+### Important Links
+
+* [Worker Repo](https://github.com/nodejs/worker)
+
+## Error Messages
+
+### Champion
+[James Snell][jasnell]
```

Comment:
I would like to be a stakeholder of the error message initiatives, thanks

Reply:
@MylesBorins ping

Reply:
I would like to be a stakeholder for the error messages as well. @joyeecheung @jasnell Is that fine for you? :-)